test/module.js

```diff
@@ -239,8 +239,8 @@ describe('dot', function () {
     });
 });
 
-describe('multiple super layers', function(){
-    it('should correctly set the order property', function(){
+describe('multiple super layers', function () {
```

Comment:
Picky opinion: I'd try to not add different stuff to the same commit. For me, this should belong to a commit like "linting?".

Reply:
ouch, a classic, don't worry 🤗 .